src/Microsoft.Data.Analysis/DataFrame.IO.cs

```diff
@@ -344,7 +344,7 @@ private static DataFrameColumn CreateColumn(Type kind, string[] columnNames, int
 
             if (encoding is null)
             {
-                encoding = Encoding.ASCII;
+                encoding = Encoding.UTF8;
```

Comment:
A unit test is failing due to this change. I wonder if we're now seeing BOM in the beginning of the stream and consequently failing the parse

Reply:
What do you think about keeping this as ASCII for now so I can get this PR(and it's follow up) in? I'll log a bug to change this to UTF8. Tracking down what is happening here might take some time. @eerhardt

Reply:
That sounds fine to me. Log a bug and fix it in a separate PR.

Reply:
Filed https://github.com/dotnet/corefxlab/issues/3003

Reply:
Alright, moving back to ASCII fixed the CI failures. Should be ready to review now